src/lib/common/serial_console_u5.cpp

```diff
@@ -167,6 +167,7 @@ static void processConsoleRxByte(void *serialHandle, uint8_t byte) {
 void serialConsoleEnable() {
   serialConsoleHandle->enabled = true;
   ulConsoleBuffIdx = 0;
+  serialConsoleHandle->flags &= ~SERIAL_TX_IN_PROGRESS;
```

Comment:
This flag needs to be cleared in order to tell the `serialGenericTx` task that the USB is not currently transmitting. If we happened to have unplugged the USB during a transmission, this flag will not have been cleared, so lets make sure when we plug back in that we clear it.